be/src/exec/vectorized/hash_join_node.cpp

```diff
@@ -510,6 +511,8 @@ pipeline::OpFactories HashJoinNode::decompose_to_pipeline(pipeline::PipelineBuil
         lhs_operators.emplace_back(std::make_shared<LimitOperatorFactory>(context->next_operator_id(), id(), limit()));
     }
 
+    lhs_operators.emplace_back(std::make_shared<ChunkAccumulateOperatorFactory>(context->next_operator_id(), id()));
```

Comment:
Should have some conditions.  and should before limit operator.

Reply:
> Should have some conditions.

Does use `ChunkAccumulateOperator`, when any following condition occurs?
- inner join or right join.
- left join, with conjuncts or runtime in filters or runtime bloom filters.

Maybe we needn't judge this? since there isn't bad effect for `ChunkAccumulateOperator`, if the output chunk is large enough for join probe.